system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -528,6 +528,50 @@
           (is (= 200 status))
           (is (= 3 revision-id)))))))
 
+(deftest delete-collection-subscription-ingest-test
+  (mock-urs/create-users (system/context) [{:username "someSubId" :password "Password"}])
+  (testing "delete a collection subscription without valid permissions"
+    (let [coll1 (data-core/ingest-umm-spec-collection
+                  "PROV1"
+                   (data-umm-c/collection
+                    {:ShortName "coll1"
+                     :EntryTitle "entry-title1"})
+                   {:token "mock-echo-system-token"})]
+      (mock-urs/create-users (system/context) [{:username "someSubId" :password "Password"}])
```

Comment:
You have this line twice, is one redundant?

Reply:
Yes it was partially redundant since the 2nd one was an identical declaration. I now removed the 2nd one and shifted the 2nd test to be nested inside the first let.

Reply:
I should have clarified, I was referring to the mock-urs/create-users call. Its been a while since I've used that command, but it seems like you'd only need the one at the start of the deftest

Reply:
You're right, that 2nd use of it was completely redundant in that case!